truffle.js

```diff
@@ -99,7 +99,7 @@ module.exports = {
       network_id: "*",
     },
     xdai: {
-      url: "https://xdai-archive.blockscout.com/",
+      url: "https://xdai.colony.io/rpc/",
```

Comment:
Are we running our own node now?

Reply:
We have been for quite some time. `xdai-archive.blockscout.com` is also now dead.